src/chapters/razor-views/static-resources.rst

```diff
@@ -0,0 +1,14 @@
+Static Resources
+================
+
+Previously, we briefly saw that when adding a view, we inherited some CSS from the ``_Layout.cshtml`` file in the ``Shared`` directory.
+While every view we add for our action methods in the controllers inherits from this template, we can also add view-specific styles and scripts.
+
+To add styles that only apply to the view we are currently working on, we can use the same ``<link>`` tag we normally do for a static HTML page.
+We can also add JS files the same way. 
+However, we want to make sure of a couple things first:
+
+#. That we are overriding any styles and scripts from ``_Layout``.
+#. That we have put custom CSS or JS in a separate ``Content`` folder.
+
+We can also put any images that we need in the separate ``Content`` folder as well.
```

Comment:
To your question (via Slack), I don't think this section requires a video. 

My only suggestion would be to add a sentence or two explaining _why_ static stuff has to go in `Content/` and how paths work in this context. For example, if we have the file `Content/mystyles.css` can we use `<link href="mystyles.css">` or is another path required?

Reply:
Fixed!